docs/installation/mac.md

```diff
@@ -96,17 +97,15 @@ To start elasticsearch as a daemonized process:
 
 ### Installing Elasticsearch with Homebrew
 
-The following directions were
-[taken from the Elasticsearch docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/brew.html).
-
 ```shell
 brew tap elastic/tap
```

Comment:
Also @rhymes do we still need this command if we not using the tap?

Reply:
we do, as we're using an older version of the tap. See the output:

```
> brew info elasticsearch-oss
elastic/tap/elasticsearch-oss: stable 7.6.0 [pinned at 7.5.2]
Distributed search & analytics engine
https://www.elastic.co/products/elasticsearch
Conflicts with:
  elasticsearch
  elasticsearch-full
/usr/local/Cellar/elasticsearch-oss/7.5.2 (707 files, 363.0MB) *
  Built from source on 2020-03-04 at 11:20:59
```

Basically what we're accomplishing here is to just downgrade the tapped version

Reply:
Okay that makes sense now. Thanks for clarifying! 💯